pkg/disttask/framework/scheduler/scheduler_manager.go

```diff
@@ -411,3 +415,96 @@ func (sm *Manager) MockScheduler(task *proto.Task) *BaseScheduler {
 		slotMgr: sm.slotMgr,
 	})
 }
+
+func (sm *Manager) collectMetricsLoop() {
+	logutil.Logger(sm.ctx).Info("collect metrics loop start")
+	ticker := time.NewTicker(defaultCollectMetricsInterval)
+	defer ticker.Stop()
+	for {
+		select {
+		case <-sm.ctx.Done():
+			logutil.BgLogger().Info("collect metrics loop exits", zap.Error(sm.ctx.Err()))
```

Comment:
```suggestion
			logutil.BgLogger().Info("collect metrics loop exits")
```
it's always cancelled